ui/imports/shared/panels/CurveSeparatorWithText.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+RowLayout {
+    id: root
+
+    property string text
+
+    Layout.preferredHeight: 32
+
+    Canvas {
+        property int cornerRadius: 9
+        property int lineWidth: 2
+        property int verticalLine: 12
+        property int horizontalLine: 19
+
+        width: horizontalLine + cornerRadius
+        height: verticalLine + lineWidth + cornerRadius
```

Comment:
And this won't work, should use `Layout.preferedWidth/Height` because you are inside a layout